### PR TITLE
Add new roles to users

### PR DESCRIPTION
**Pull request by LimberHope**
add new roles for users, project_developer, funder, government and new fields (country, program) to the users table
[Task](https://gfw.atlassian.net/browse/TM-599)

### CODE REVIEW

File: app/Console/Commands/Migration/RolesMigrationCommand.php [hunk 1]
```diff
@@ -28,7 +28,7 @@ class RolesMigrationCommand extends Command
     public function handle()
     {
         if ($this->option('log')) {
-            echo('* * * Started * * * ' . $this->description . chr(10));
+            echo ('* * * Started * * * ' . $this->description . chr(10));
```
Comment (roguenet) [inline app/Console/Commands/Migration/RolesMigrationCommand.php:31]: The linter hasn't completed as of this review, bug I suspect it'll complain about this change. I don't see why this space is needed.

File: app/Console/Commands/Migration/RolesMigrationCommand.php [hunk 2]
```diff
@@ -88,7 +103,7 @@ public function handle()
             });
 
         if ($this->option('log')) {
-            echo('- - - Finished - - - ' . chr(10));
+            echo ('- - - Finished - - - ' . chr(10));
```
Comment (roguenet) [inline app/Console/Commands/Migration/RolesMigrationCommand.php:106]: same as above with this change.

File: app/functions.php [hunk 3]
```diff
@@ -154,7 +154,7 @@ function assignSpatieRole($user)
 {
     switch($user->role) {
         case 'user':
-            $user->assignRole('project-developer');
+            $user->assignRole('project_developer');
```
Comment (roguenet) [inline app/functions.php:157]: Is this change intentional? This seems like it might have some pretty serious downstream consequences; changing the name of this role.

File: app/Console/Commands/Migration/RolesMigrationCommand.php (hunk 1 above)
Comment (pachonjcl) [reply app/Console/Commands/Migration/RolesMigrationCommand.php]: space removed

File: app/Console/Commands/Migration/RolesMigrationCommand.php (hunk 2 above)
Comment (pachonjcl) [reply app/Console/Commands/Migration/RolesMigrationCommand.php]: same

File: app/functions.php (hunk 3 above)
Comment (pachonjcl) [reply app/functions.php]: changing it back to use dash instead of underscore

File: app/Policies/Policy.php [hunk 4]
```diff
@@ -37,6 +37,13 @@ protected function isAdmin(?UserModel $user): bool
         return ! $this->isGuest($user) && $user->role == 'admin';
     }
 
+    protected function isNewRoleUser(?UserModel $user): bool
+    {
+        $newRoles = ['project_developer', 'funder', 'government', 'user'];
```
Comment (roguenet) [inline app/Policies/Policy.php:42]: Should this be `project-developer`?

Comment (pachonjcl) [reply app/Policies/Policy.php]: yes, changed.

**Review by roguenet — DISMISSED** [review]

**Review by roguenet — APPROVED** [review]
